extensions/flyway/deployment/src/main/java/io/quarkus/flyway/deployment/FlywayBuildItem.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.flyway.deployment;
+
+import org.jboss.jandex.AnnotationInstance;
+
+import io.quarkus.builder.item.MultiBuildItem;
+import io.quarkus.flyway.runtime.FlywayContainerProducer;
+import io.quarkus.flyway.runtime.FlywayDataSourceBuildTimeConfig;
+
+public final class FlywayBuildItem extends MultiBuildItem {
```

Comment:
I would very much like to have some Javadoc on this.

Reply:
@geoand, just updated it !

Reply:
Thanks!